src/group_theory/subgroup.lean

```diff
@@ -619,6 +619,47 @@ end
 lemma closure_singleton_one : closure ({1} : set G) = ⊥ :=
 by simp [eq_bot_iff_forall, mem_closure_singleton]
 
+@[to_additive]
```

Comment:
Can someone check the two `@[to_additive]`? I am not sure this is the correct way of using it (I can of course just prove the additive version by hand).

Reply:
They both look fine to me